sdk/keyvault/Azure.Security.KeyVault.Secrets/tests/SecretClientLiveTests.cs

```diff
@@ -0,0 +1,265 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using Azure.Security.KeyVault.Secrets;
+using Azure.Core.Testing;
+
+namespace Azure.Security.KeyVault.Test
+{
+    public class SecretClientLiveTests : KeyVaultTestBase
+    {
+        private const int PagedSecretCount = 50;
```

Comment:
Is this the cap to where paging starts?

Reply:
It's what we used before, I'm going to verify that we hit multiple pages.

Reply:
anything over 25 should hit multiple pages.  That is the current default page max for the service.  However this could change, so perhaps it would be nice if we had a way to verify that we hit multiple pages.

Reply:
@pakrym [here](https://microsoft.sharepoint.com/teams/AzureDeveloperExperience/_layouts/OneNote.aspx?id=%2Fteams%2FAzureDeveloperExperience%2FShared%20Documents%2FService%20-%20KeyVault%2FAzure%20SDK%20-%20Service%20-KeyVault&wd=target%28Code%20Sources.one%7C07F69742-3F61-42AF-A575-F09A5E201965%2FTests%7C5F230B73-61CA-46C8-B13E-DF9EEC420AF8%2F%29) are the scenarios we want to cover for all languages. 
For list, we agree that we could just ask for 2 - 3 values for all list test, and have one specific one that will test the paging.
For the remaining scenarios that haven't been cover, I could add them later.

Reply:
Why not force paging in all list operations?

Reply:
Preferences? I mean, why not only on one if at the end they all call the same paging operation?
The goal is to test that each invocation works. Other test can test that paging works.
If paging implementation varies per method, then yeah, make sure paging works in all scenarios